msm/src/fec/mod.rs

```diff
@@ -35,14 +38,17 @@ mod tests {
         rng: &mut RNG,
         domain_size: usize,
     ) -> FECWitnessBuilderEnv {
-        use mina_curves::pasta::{Fp, Pallas};
+        // NOTE: this uses Pallas-in-Pallas emulation.
+        use mina_curves::pasta::Pallas;
+        type Fp = <Pallas as AffineRepr>::ScalarField;
```

Comment:
The scalar field of Pallas is `Fq`: https://github.com/o1-labs/proof-systems/blob/master/curves/src/pasta/curves/pallas.rs#L13

Reply:
Renamed :+1: